HW-0.Rmd

```diff
@@ -1,14 +1,14 @@
 ---
 title: "Testing"
-author: "WRITE YOUR NAME HERE"
-date: "February 3, 2016"
+author: "Sanket Vadlamani"
+date: "February 23, 2016"
 output: html_document
 ---
-
-Your tasks!
-
-* Change the name of the author above to yours
+  
+  Your tasks!
+  
+  * Change the name of the author above to yours
 * Change the date
-* Replace the NHL URL with the URL for the document you just posted to Rpubs **here**: [My RPubs Document](https://www.nhl.com/)
+* Replace the NHL URL with the URL for the document you just posted to Rpubs **here**: [My RPubs Document](http://rpubs.com/uvadlamani/hw00)
 * Click Knit HTML
```

Comment:
Looks good. And I love the arbitrary title.

Rpubs is a good way for people to share R code/output on the web for collaboration. Especially useful since you can iterate quickly.